rolldice: add page (#5811)
tldr-pages/tldr: pages/linux/rolldice.md

# rolldice

> Roll virtual dice.

> - Roll a single 20 sided die:

Reviewer: ```suggestion
- Roll a single 20 sided dice:
```
As far as I know, `dice` is more common than `die`.

> `rolldice d20`

Reviewer: ```suggestion
`rolldice d{{20}}`
```
We keep any variables in `{{ }}`

> - Roll two six sided dice and keep the highest roll:

Reviewer: ```suggestion
- Roll two six sided dice and drop the lowest roll:
```
I think that explains better what `s1` does.

> `rolldice 2d6s1`

Reviewer: ```suggestion
`rolldice {{2}}d{{6}}s{{1}}`
```

- Roll two 20 sided dice and add a modifier value:

> `rolldice 2d20+5`

Reviewer: ```suggestion
`rolldice {{2}}d{{20}}{{+5}}`
```

# rolldice

> Roll virtual dice.
> More information: <https://manned.org/rolldice>.

- Roll a single 20 sided dice:

`rolldice d{{20}}`

- Roll two six sided dice and drop the lowest roll:

`rolldice {{2}}d{{6}}s{{1}}`

- Roll two 20 sided dice and add a modifier value:

`rolldice {{2}}d{{20}}{{+5}}`

> - Roll a d20 two times:

Reviewer: ```suggestion
- Roll a 20 sided dice two times:
```
